lib/utils/ripple_page_route.dart

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:flutter/material.dart';
+
+class RipplePageRoute extends PageRouteBuilder {
+  RipplePageRoute({required this.position, required this.child})
+    : super(
+        transitionDuration: const Duration(milliseconds: 700),
+        pageBuilder: (_, __, ___) => child,
```

Comment:
I think the unnamed parameters can now all be single underscores even when there is more than one parameter, so this could be `pageBuilder: (_, _, _) => child,`.